Documentation/contributing/development/contributing_guide.rst

```diff
@@ -100,6 +100,17 @@ requirements have been met:
       the first 12 characters of the git SHA followed by the full commit title
       as seen above without breaking the line.
 
+#. If you change CLI arguments of any binaries in this repo, the CI will reject your PR if you don't
+also update the command reference docs. To do so, make sure to build the binaries, run the relevant script
+and commit the changes:
+
+.. code-block:: shell-session
+
+    $ make daemon bugtool operator cilium cilium-health
+    $ make -C Documentation update-cmdref
+    $ git add Documentation/cmdref
+    $ git commit
```

Comment:
Ideally squash with the commit that introduced the changes, but not sure how to write that simply if the author is not familiar with amending/squashing :confused:. Probably easier to keep it this way I guess.

Reply:
I didn't really want to go into the details of how to commit here :/, but I am also not 100% happy with this

Reply:
`make` or `make postcheck` will achieve the same as the two make steps here (cmdref update is part of the default target).

Reply:
@joestringer thanks! changed it out to `make postcheck`